lib/matplotlib/path.py

```diff
@@ -281,7 +282,14 @@ def __deepcopy__(self, memo=None):
         readonly, even if the source `Path` is.
         """
         # Deepcopying arrays (vertices, codes) strips the writeable=False flag.
-        p = copy.deepcopy(super(), memo)
+        if memo is None:
+            memo = {}
```

Comment:
This is not needed, `memo` is always a dict.

The default value for `memo` is also not needed. The `memo` argument is always passed to `__deepcopy__()`.

Reply:
Yes, when called via `copy.deepcopy`, but this `__deepcopy__` method is also aliased to `deepcopy` just below the GitHub patch context

Reply:
we could change our deepcopy to call `copy.deepcopy` internally (which might bet better?)